dart/lib/src/sentry_options.dart

```diff
@@ -323,6 +329,9 @@ class SentryOptions {
   @internal
   late SentryStackTraceFactory stackTraceFactory =
       SentryStackTraceFactory(this);
+
+  bool get isNativeIntegrationAvailableAndEnabled =>
```

Comment:
Since `enableNative` is going to be removed, we can keep using`options.platformChecker.hasNativeIntegration`